### PR TITLE
chore(avatar): update avatar svg

### PR DESCRIPTION
**Pull request by andrew-ronaldson**
Closes https://github.com/patternfly/patternfly/issues/6758
Minor change to the svg for the avatar component to align with the v6 Figma library.

### CODE REVIEW

**Review by bekah-stephens — APPROVED** [review]

File: src/patternfly/assets/images/img_avatar-dark.svg [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-</style>
-<rect class="st0" width="36" height="36"/>
-<path class="st1" d="M30.5,36c-0.4-3.9-1.3-9-2.9-11c-1.1-1.4-2.3-2.2-3.5-2.6s-1.8-0.6-6.3-0.6s-6.1,0.7-6.1,0.7
-	c-1.2,0.4-2.4,1.2-3.4,2.6C6.7,27,5.8,32.2,5.4,36H30.5z"/>
-<path class="st1" d="M17.7,20.1c-3.5,0-6.4-2.9-6.4-6.4s2.9-6.4,6.4-6.4s6.4,2.9,6.4,6.4S21.3,20.1,17.7,20.1z"/>
-<path class="st2" d="M13.3,36l0-6.7c-2,0.4-2.9,1.4-3.1,3.5L10.1,36H13.3z"/>
-<path class="st2" d="M22.7,36l0-6.7c2,0.4,2.9,1.4,3.1,3.5l0.1,3.2H22.7z"/>
-</svg>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width=".5in" height=".5in" version="1.1" viewBox="0 0 36 36">
```
Comment (srambach) [inline src/patternfly/assets/images/img_avatar-dark.svg:2]: These have a width and height on the svg but the old ones did not - I'm not sure if that's good or bad or matters?

Comment (srambach) [reply src/patternfly/assets/images/img_avatar-dark.svg:2]: It's possibly affecting https://patternfly-pr-6769.surge.sh/components/menus/menu-toggle#with-avatar-and-text

Comment (mcoker) [reply src/patternfly/assets/images/img_avatar-dark.svg:2]: Nice catch, I'd remove that. Pretty sure everywhere we use an avatar, we constrain the size if that's needed.

**Review by mcoker — APPROVED** [review]
LGTM if we can remove the width/height.

**Pull request merged**